.github/dependabot.yml

```diff
@@ -9,6 +9,8 @@ updates:
     schedule:
       interval: "weekly"
     ignore:
+      - dependency-name: "idna_adapter"
+        # stay with <= 1.2.0 for Rust 1.78
       - dependency-name: "litemap"
         # stay with <= 0.7.4 for Rust 1.76
```

Comment:
MSRV is 1.78, is the `litemap` ignore still correct?

Reply:
According to [comments](https://github.com/Azure/azure-init/blob/628d80f2c012f8b762c51f5a7cf090a879fe8677/libazureinit/Cargo.toml#L29), they should be kept until Sep. 2025.